export/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/CStoreSender.java

```diff
@@ -80,7 +80,7 @@ public void send(PubsubMessage message) throws Exception {
     }
 
     CountingInputStream countingStream = new CountingInputStream(part.getInputStream());
```

Comment:
Can you please remove the old logic in this file which gets transferSyntaxUid (and passes it as an include field)?

Reply:
Ok